torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -192,6 +200,13 @@ def permute_hidden(self, hx, permutation):
             return hx
         return apply_permutation(hx, permutation)
 
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
```

Comment:
We need to ensure that the test_backward_compatibility tests check for versioning for serialization. Can you please add a test for that?

Reply:
Sure, I can do that in a follow up PR. Adding serialized `.pt` files doesn't work with the ghimport flow so I'll have to create a different PR for it.